test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorWebSite.TagHelperComponent.Body.html

```diff
@@ -1,3 +1,4 @@
 <body inject="true">
 Hello from Body Tag Helper Component
-<script>'This was injected!!'</script></body>
\ No newline at end of file
+<script>'This was injected!!'</script>
+</body>
```

Comment:
Confusing that this and RazorWebSite.TagHelperComponent.Head.html changed at all. As mentioned later, revert the .cshtml changes.